packages/dashboard/src/components/tests/test-utils.tsx

```diff
@@ -0,0 +1,27 @@
+import { act, render, RenderResult } from '@testing-library/react';
+import React from 'react';
+import { User, UserContext } from '../auth/contexts';
+
+/**
+ * Wraps a `render` in a `act`.
```

Comment:
This is just when we have async elements, right?  not for every case

Reply:
I think this function is a mistake from when I don't understand how `@testing-library` works. `render` should already be automatically wrapped in `act` and we should use `waitFor` or `waitForElementToBeRemoved` to handle async events.

Reply:
> `render` should already be automatically wrapped in `act` and we should use `waitFor` or `waitForElementToBeRemoved` to handle async events.

Yeah, IIRC this is how it works